packages/react/src/components/tabs/tabs.tsx

```diff
@@ -139,11 +154,18 @@ export const Tabs: VoidFunctionComponent<Props> = ({
                         isSelected={isTabSelected(tabItem.id)}
                         ref={tabItem.buttonRef}
                         onClick={() => handleTabSelected(tabItem)}
+                        onDelete={tabItem.onDelete}
                         onKeyDown={(event) => handleButtonKeyDown(event, tabItem)}
                     >
                         {tabItem.title}
                     </TabButton>
                 ))}
+                {onAddTab && (
+                    <Button buttonType='tertiary' type="button" onClick={onAddTab}>
+                        <AddIcon name="plusSign" size='16' aria-hidden="true" focusable={false} />
+                        {t('addTabs')}
```

Comment:
Le bouton sert à ajouter un seul tab
```suggestion
                        {t('addTab')}
```